resources/home/dnanexus/generate_workbook/tests/test_columns.py

```
@@ -1,14 +1,14 @@
-import argparse
 import os
 import subprocess
 import sys
-
+from unittest.mock import patch
 import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)